src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -967,6 +967,13 @@ std::string ContextAttribute::check
   size_t len;
   char errorMsg[128];
 
+  if (((ciP->apiVersion == "v2") && (len = strlen(name.c_str())) < MIN_ID_LEN) && (requestType != EntityAttributeValueRequest))
+  {
+    snprintf(errorMsg, sizeof errorMsg, "attribute name length: %zd, min length supported: %d", len, MIN_ID_LEN);
+    alarmMgr.badInput(clientIp, errorMsg);
+    return std::string(errorMsg);
+  }
```

Comment:
> Strange ... the modified functest had that string. Perhaps 'name' is inserted via %s or streams or something ...

Correctly, `had`. Note the "in the PR branch". I mean, after the changes I did in .test and code, no occurrences.

Let's see now for the other pattern you suggest...